packages/module/src/ConsoleInternal/components/catalog/_catalog.scss

```diff
@@ -103,7 +103,7 @@ $catalog-tile-width: $pfext-m-catalog-tile-width;
 
 .pfext-catalog-page {
   background: $color-pf-white;
-  border: 1px solid var(--pf-v5-global--BorderColor--300);
+  border: 1px solid var(--pf-v6-global--BorderColor--300);
```

Comment:
We have a bunch of border color and border width tokens that should probably be used here, you can search for "global--border--color" and "global--border--width" on https://github.com/patternfly/patternfly/blob/v6/src/patternfly/base/tokens/tokens-default.scss

I try to use `--pf-t--global--border--color--default` wherever possible for the color, and if this is just a regular kind of element with a border and isn't interactive or anything, for the border width `--pf-t--global--border--width--regular` or `--pf-t--global--border--width--box--default` would probably be appropriate.